engine/dlib/src/test/test_httpserver.cpp

```diff
@@ -136,24 +137,30 @@ class dmHttpServerTest: public jc_test_base_class
         self->m_ClientData.append((const char*) content_data, content_data_size);
     }
 
+#define T_ASSERT_LE(_A, _B) \
+    if ( !((_A) < (_B)) ) { \
+        printf("%s:%d: ASSERT: %s < %s: %d < %d", __FILE__, __LINE__, #_A, #_B, (int)(_A), (int)(_B)); \
+    } \
+    assert( (_A) < (_B) );
+
```

Comment:
`jc_test.h` doesn't have thread protection (not yet at least), so we cannot use the regular test asserts in the thread.